source/blog/2016-12-20-ci-please-build.html.md

```diff
@@ -0,0 +1,44 @@
+---
+title: CI Please Build&mdash;How to build your oVirt project on-demand
+author: eedri
+tags: infrastructure, developers, ci
+date: 2016-12-20 10:00:00 CET
+---
+
+All projects in oVirt CI are built today post merge, using the 'build-artifacs' stage from [oVirt's CI standards](http://ovirt-infra-docs.readthedocs.io/en/latest/CI/Build_and_test_standards.html).
+
+This ensures that all oVirt projects are built and deployed to oVirt repositories and can be consumed by CI jobs, developers or oVirt users.
+However, on some occasions a developer might need to build his project from an open patch in order to test the code before it is merged to the branch,
+So in any case of errors, he can go back to the patch and fix them before merging.
+Another exmaple is a UI developer, that works on a new UI fix/feature and wants to see the changes on each patch change.
+
+## The Current Build Option
+
+Until now, to build rpms from a patch, a developer needed to use a custom [Jenkins job](http://jenkins.ovirt.org/job/ovirt-engine_master_build-artifacts-el7-x86_64_build_from_patch/), which was only available to ovirt-engine and only for master branch. 
+Another option was to try and build it locally, but then you need to make sure you have all the right packages installed ( something that is given automatically in CI ).
```

Comment:
Do you think its worth adding it to the blog? my guess most developer don't use it anyhow